### ISSUE TITLE
Disable pause on death

### ISSUE THREAD

**Issue by Kaotic07** [EasyFarm/EasyFarm#324]
Is there any way to disable the pause on death that the program defaults to? It would be awesome if there was a simple option near "HP on Death" or if there was something I was missing that would allow me to do this. Thanks!

**Comment by Mercurian** [comment]
Hp on death already exists

**Comment by Kaotic07** [comment]
Yeah but the program pauses when you die, my goal is to be able to raise up and continue what I was doing.

**Comment by Mykezero** [comment]
This feature can be added back in as an additional options setting.

**Comment by Sinnersslayer** [comment]
The auto-pause option is good then you die, HP and stay in the city.
It prevents bot from trying to cast Trusts or buffs which can't be casted in the city like GEO Indi spells.

**Comment by maven79** [comment]
What ever happened to this? And all you have to do is check if you are in a city and then "pause when in town" then you don't have to worry about all the running into walls nonsense.
